firefly_client/firefly_client.py

```diff
@@ -60,7 +60,7 @@ class FireflyClient(WebSocketClient):
         Show a fits image.
     show_table(file_on_server, tbl_id, title, page_size, is_catalog)
         Show a table.
-    show_xyplot(file_on_server, **additional_params)
+    show_xyplot(file_on_server, standalone, chart_params)
```

Comment:
In this docstring, please change `chart_params` to `**chart_params`. Python users will recognize that these are optional keyword arguments.